src/components/claims/claim-details/components/ClaimInformation.tsx

```diff
@@ -161,9 +155,17 @@ export const ClaimInformation: React.FC<{
             enumToSelectFrom={ClaimState}
             placeholder={''}
             onChange={async (value) => {
-              await updateClaimState(
-                updateClaimStateOptions(claimId, validateSelectOption(value)),
-              )
+              await updateClaimState({
+                variables: { id: claimId, state: validateSelectOption(value) },
+                optimisticResponse: {
+                  updateClaimState: {
+                    id: claimId,
+                    __typename: 'Claim',
+                    state: validateSelectOption(value),
+                    events: [],
```

Comment:
This isn't super critical, but the (more) correct way would be to do something like 
```suggestion
                    events: claimInformationData?.claim?.events ?? [],
```

Since this won't empty the `events` cache while we're waiting for the actual response. This leaves the `events` field intact while we're waiting for the mutation to actually succeed!